Application/EdFi.Ods.Api/Security/AuthorizationStrategies/Relationships/RelationshipsAuthorizationStrategyBase.cs

```diff
@@ -141,6 +147,7 @@ string GetAuthorizationFilterName(string subjectEndpointName, string authorizati
         }
 
         private readonly ConcurrentDictionary<(string, string), string> _filterNameBySubjectAndPathModifier = new();
+        private readonly ConcurrentDictionary<string, string> _nonRoleNamedByRoleNamedPropertyName = new();
```

Comment:
Let's rename this to `_nonRoleNamedByRawPropertyName` because otherwise it reads like the key is a role-named property, but really it's more of a _possibly_ role named property. I think "raw" property name is a better term here.

Reply:
Renamed